hoomd_organics/base/system.py

```diff
@@ -150,6 +150,11 @@ def mass(self):
             )
         return sum(mol.mass for mol in self.all_molecules)
 
+    @property
+    def net_charge(self):
+        if self.gmso_system:
```

Comment:
I wonder if we still need this `if self.gmso_system` anymore? It's used in `n_particles` and mass as well. I think with the recent changes, `self.gmso_system` will always exist.

Reply:
So, I think we need it for the `mass` attribute, since that is used by `set_target_box` before `self.gmso_system` is created, but we probably don't need it in the `n_particles` and `net_charge` attributes.  I'll test it out and push more changes if so.